main.tf

```
@@ -0,0 +1,67 @@
+module "label" {
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.16.0"
+  enabled    = var.enabled
+  namespace  = var.namespace
+  name       = "${var.name}-ses"
+  stage      = var.stage
+  delimiter  = var.delimiter
+  attributes = var.attributes
+  tags       = var.tags
+}
+
+/*
+Create SES domain identity and verify it with Route53 DNS records
+*/
+
+resource "aws_ses_domain_identity" "ses_domain" {
+  domain = var.domain
+}
+
+resource "aws_route53_record" "amazonses_verification_record" {
+  count = var.verify_domain ? 1 : 0
```

```suggestion
  count = var.enabled && var.verify_domain ? 1 : 0
```

Since we provide the `enabled` variable, all resources should be controlled by it (not only the label)